deploy_comment.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+import logging
+import os
+import sys
+
+from utils import notify_irc, parse_link_headers
+
+import requests
+
+log = logging.getLogger()
+handler = logging.StreamHandler(sys.stderr)
+log.addHandler(handler)
+log.setLevel(logging.DEBUG)
+
+logging.getLogger("requests").propagate = False
+
+GITHUB_USERNAME = os.getenv('GITHUB_USERNAME')
+GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
+
+log.info('Running deploy_comment.py: {}'.format(datetime.datetime.now()))
+repos = 'https://api.github.com/repos/'
+myself = 'addons-robot'
+
+message = ('This has been deployed to our '
+'[development server](https://addons-dev.allizom.org). Please see '
+'[push duty](https://addons.readthedocs.io/en/latest/server/push-duty.html) '
+'for when it will be deployed to production.')
```

Comment:
I actually had code to check the addons-calendar for when scheduled deployments will happen. Problems exist for this code around tagging and stuff, so I'm not sure if we should aim for this yet.